### ISSUE TITLE
the name `AsyncWrite` is defined multiple times

### ISSUE THREAD

**Issue by andrewbanchich** [nats-io/nats.rs#74]
#### natsio version (`grep 'name = "natsio"' Cargo.lock -A 1`)

BTW, I think the above command needs to be updated to use "nats" instead of "natsio".

version = "0.6.0"

#### rustc version (`rustc --version` - we support Rust 1.37 and up)

rustc 1.44.1 (c7087fe00 2020-06-17)

#### OS/Container environment:

macOS Catalina 10.15.5

When I add `nats` to my project I get several errors:

```
error[E0252]: the name `AsyncWrite` is defined multiple times
 --> /Users/andrewbanchich/.cargo/registry/src/github.com-1ecc6299db9ec823/async-native-tls-0.3.3/src/runtime.rs:9:53
  |
2 | pub(crate) use async_std::io::{Read as AsyncRead, Write as AsyncWrite};
  |                                                   ------------------- previous import of the trait `AsyncWrite` here
...
9 | pub(crate) use tokio::io::{AsyncRead, AsyncReadExt, AsyncWrite};
  |                                                     ^^^^^^^^^^ `AsyncWrite` reimported here
  |
  = note: `AsyncWrite` must be defined only once in the type namespace of this module
help: you can use `as` to change the binding name of the import
  |
9 | pub(crate) use tokio::io::{AsyncRead, AsyncReadExt, AsyncWrite as OtherAsyncWrite};
```

```
error: only one of 'runtime-async-std' or 'runtime-tokio' features must be enabled
  --> /Users/andrewbanchich/.cargo/registry/src/github.com-1ecc6299db9ec823/async-native-tls-0.3.3/src/lib.rs:38:1
   |
38 | compile_error!("only one of 'runtime-async-std' or 'runtime-tokio' features must be enabled");
```

```
error[E0407]: method `poll_shutdown` is not a member of trait `AsyncWrite`
   --> /Users/andrewbanchich/.cargo/registry/src/github.com-1ecc6299db9ec823/async-native-tls-0.3.3/src/tls_stream.rs:116:5
    |
116 | /     fn poll_shutdown(mut self: Pin<&mut Self>, ctx: &mut Context<'_>) -> Poll<io::Result<()>> {
117 | |         match self.with_context(ctx, |s| s.shutdown()) {
118 | |             Ok(()) => Poll::Ready(Ok(())),
119 | |             Err(ref e) if e.kind() == io::ErrorKind::WouldBlock => Poll::Pending,
120 | |             Err(e) => Poll::Ready(Err(e)),
121 | |         }
122 | |     }
    | |_____^ not a member of trait `AsyncWrite`
```

This is only after adding `nats` as a dependency; I'm not using it in my code anywhere yet.

I'm currently using the `smol` runtime (which is awesome by the way).

**Comment by stjepang** [comment]
This is going to be fixed by #75, which replaces the offending `async-native-tls` crate with `async-tls`.

**Comment by andrewbanchich** [comment]
Great, thanks!

**Comment by stjepang** [comment]
Fixed in v0.7.0 :)
